doc/src/sgml/ref/create_materialized_view.sgml

```diff
@@ -58,8 +58,79 @@ CREATE [ INCREMENTAL ] MATERIALIZED VIEW [ IF NOT EXISTS ] <replaceable>table_na
     <term><literal>INCREMENTAL</literal></term>
     <listitem>
      <para>
-      If specified, a materialized view enables incremental view maintenance.
-      You can replace only the contents of a materialized view, which based rows are changed.
+      If specified, some triggers and other objects are automatically created
+      so that the rows of the materialized view is immediately updated when
```

Comment:
I am not sure what "other objects" refer. Is there other object created other than triggers?

I think "the rows of the materialized view is" would be "the rows of the materialized view are"

Reply:
I thought there might be an use of intermediate tables in future releases. If we do not have such a plan (at least by the end of this year), "other objects" should be removed.